psp/data_sources/nwp.py

```diff
@@ -1,72 +1,20 @@
 import datetime as dt
+import logging
 import pathlib
 import pickle
-from typing import Optional, TypeVar
+from typing import Optional
 
 # This import registers a codec.
 import ocf_blosc2  # noqa
 import xarray as xr
 
+from psp.data_sources.utils import _STEP, _TIME, _VALUE, _VARIABLE, _X, _Y, slice_on_lat_lon
```

Comment:
Nice, have you moved this just to keep things cleaner?

Reply:
yea